src/main/java/com/florianmski/suncalc/utils/Constants.java

```diff
@@ -2,35 +2,74 @@
 
 public class Constants
 {
+    /** units: radians/degrees */
     public final static double RAD = Math.PI / 180.0;
-    public final static double E = RAD * 23.4397; // obliquity of the Earth
 
+    /** obliquity of the Earth */
+    public final static double E = RAD * 23.4397;
+
+    /**
+     * <p>
+     *     Solar elevation angles in degrees, i.e. the angle of the sun from the horizon.
+     *     Defined as 90 deg - solar zenith angle.
+     * </p>
+     * <p>
+     *     See <a href="https://en.wikipedia.org/wiki/Solar_zenith_angle">
+     *     https://en.wikipedia.org/wiki/Solar_zenith_angle</a> and
+     *     <a href="https://en.wikipedia.org/wiki/Twilight">https://en.wikipedia.org/wiki/Twilight</a>
+     *     for more info.
+     * </p>
+     */
     public class SunAngles
     {
+        /** sunrise (top edge of the sun appears on the horizon) */
         public final static double SUNRISE_START                        = -0.833;
-        public final static double GOLDEN_HOUR_MORNING_START            = -0.3;
+        /** morning golden hour starts (soft light, best time for photography) */
+         public final static double GOLDEN_HOUR_MORNING_START            = -0.3;
+        /** daylight starts */
```

Comment:
So i basically was inspired by the original https://github.com/mourner/suncalc#reference section and thought it would be helpful to add these Javadocs.

Do you think they add enough value? I can possible improve them further